force-app/main/default/lwc/apiSetterGetter/apiSetterGetter.html

```diff
@@ -15,16 +15,16 @@
                 onchange={handlePriorityChange}
                 checked={priority}
             ></lightning-input>
-            <lightning-button
-                label="Add Todo"
-                variant="brand"
-                onclick={handleSave}
-                class="slds-show slds-var-p-vertical_small"
-            ></lightning-button>
-            <c-todo-list
-                class="slds-show slds-is-relative"
-                todos={todos}
-            ></c-todo-list>
+            <div class="slds-var-p-vertical_small">
+                <lightning-button
+                    label="Add Todo"
+                    variant="brand"
+                    onclick={handleSave}
+                ></lightning-button>
+            </div>
+            <div class="c-todo-list slds-is-relative">
```

Comment:
Is `slds-is-relative` still needed?

Reply:
Yep. There's a CSS-based label that calls attention to the component composition. Without `slds-is-relative` it becomes displaced.

![image](https://user-images.githubusercontent.com/1127876/98560010-35a3be00-229f-11eb-9b41-2d476e6c6354.png)

Here's what it should look like: 

![image](https://user-images.githubusercontent.com/1127876/98560075-4b18e800-229f-11eb-9542-3a680ed730ac.png)